model.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+
+	"github.com/filecoin-project/sentinel-tick/quotetracker"
+	pg "github.com/go-pg/pg/v10"
+)
+
+const schema = "filquotes"
+
+// Quote stores FIL price information.
+type Quote struct {
+	//lint:ignore U1000 hit for go-pg
+	tableName struct{} `pg:"filquotes.fil_usd_quotes"`
```

Comment:
I think this was where @placer14 was suggesting using a version suffix, e.g. `fil_usd_quotes_v1`

Reply:
Also table is called fil_usd_quotes but has a currency column?